ludwig/encoders/set_encoders.py

```diff
@@ -91,22 +91,28 @@ def __init__(
             default_weights_regularizer=weights_regularizer,
             default_bias_regularizer=bias_regularizer,
             default_activity_regularizer=activity_regularizer,
-            # default_weights_constraint=weights_constraint,
-            # default_bias_constraint=bias_constraint,
             default_norm=norm,
             default_norm_params=norm_params,
             default_activation=activation,
             default_dropout=dropout,
         )
 
-    def call(self, inputs, training=None, mask=None):
+    def forward(self, inputs, training=None, mask=None):
         """
             :param inputs: The inputs fed into the encoder.
                    Shape: [batch x 1], type tf.int32
 
             :param return: embeddings of shape [batch x embed size], type tf.float32
         """
-        hidden = self.embed_sparse(inputs, training=training, mask=mask)
+        hidden = self.embed(inputs, training=training, mask=mask)
         hidden = self.fc_stack(hidden, training=training, mask=mask)
 
         return hidden
+
+    @property
+    def input_shape(self) -> torch.Size:
+        return torch.Size([1])
```

Comment:
Is 1 correct here?

Reply:
I can double check

Reply:
I believe so -- set encoders pass the input directly to the `Embed` module, so they should have the same input shape